js/hitbtc3.js

```diff
@@ -302,6 +302,15 @@ module.exports = class hitbtc3 extends Exchange {
                     'future': 'derivatives',
                     'derivatives': 'derivatives',
                 },
+                'setLeverage': {
+                    'maxLeverage': {
+                        'BTC/USDT:USDT': 100,
+                        'ETH/USDT:USDT': 75,
+                        'ADA/USDT:USDT': 75,
+                        'SOL/USDT:USDT': 75,
+                        'XRP/USDT:USDT': 75,
+                    },
```

Comment:
You can get this information from `this.market (symbol)['limits']['leverage']['max']` so don't hard code it, also, I counted 23 swap markets in `fetchMarkets`, I don't know if that matters

Reply:
@samgermain as far as I understood from the code, the intent was to override the max leverage for some of them, while all others default to 50, but that might need one more confirmation, indeed. @Dan-krm could you please rewrite once again it to use the max leverage limit from the market structure as @samgermain suggested?

Reply:
@Dan-krm nvm, i'll add this edit.

Reply:
Thanks for adding the edit, I'll keep this approach in mind for the future